formats/cbor/commonMain/src/kotlinx/serialization/cbor/Cbor.kt

```diff
@@ -322,6 +364,19 @@ public class Cbor(
             return result
         }
 
+        /**
+         * Reads the specified number of bytes from the input stream and returns a [ByteArray]
+         * containing the read bytes.
+         *
+         * @throws [SerializationException] if the provided length does not fit in a 32-bit signed [Int].
```

Comment:
Brackets not needed after `@throws`.

```suggestion
         * @throws SerializationException if the provided length does not fit in a 32-bit signed [Int].
```